legacy/src/webhooks/discord.ts

```
@@ -0,0 +1,365 @@
+        if (templateData.status) {
+          const statusIcon = this.getStatusIcon(templateData.status);
+          // Option 1: Prepend icon to title
+          // embed.title = `${statusIcon} ${embed.title}`.substring(0, 256); 
+          // Option 2: Add as a field
+           const field = createField("Status", `${statusIcon} ${templateData.status}`, true);
+           if (field) embed.fields.unshift(field); // Add to beginning
+        }
+        break;
+
+      case 'progress':
+        let percent = -1;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)